log.go

```diff
@@ -135,44 +145,58 @@ type EventLogger interface {
 	//		"type": bizStruct,
 	//      "request": req,
 	//		})
+	// Deprecated: Stop using go-log for event logging
 	SetTags(ctx context.Context, tags map[string]interface{})
 
 	// SerializeContext takes the SpanContext instance stored in `ctx` and Seralizes
 	// it to bytes. An error is returned if the `ctx` cannot be serialized to
 	// a bytes array
+	// Deprecated: Stop using go-log for event logging
 	SerializeContext(ctx context.Context) ([]byte, error)
 }
 
 // Logger retrieves an event logger by name
-func Logger(system string) EventLogger {
+func Logger(system string) *eventLogger {
 
 	// TODO if we would like to adjust log levels at run-time. Store this event
 	// logger in a map (just like the util.Logger impl)
 	if len(system) == 0 {
 		setuplog := getLogger("setup-logger")
-		setuplog.Warning("Missing name parameter")
+		setuplog.Warn("Missing name parameter")
```

Comment:
nit: Given the current configuration we will miss this message due to its level [defined here](https://github.com/ipfs/go-log/pull/61/files#diff-17a9f1db2982c66c910a1e06f92e6489R82)